README.md

```diff
@@ -57,7 +57,7 @@ Quoting the glob like `--ignore '*.md'` passes it through unexpanded and ignores
 
 #### Globbing examples
 
-To lint all Markdown files in a Node.js project (excluding dependencies), the following commands might be used:
+To lint all Markdown files recursively in a Node.js project (excluding dependencies), `*/**.md` wildcard comes in handy. The following commands might be used:
```

Comment:
I don't think `*/**.md` is what's intended? And `**/*.md` is already shown on the next two lines.

Reply:
@DavidAnson my intention was to add mention the explanation of this wildcard that is used in the code

Reply:
I think lines 62 and 64 already show the preferred syntax?

Reply:
@DavidAnson yeah, I agree it shows the syntax. 
So I didn't change the syntax, but rather add a precise clarification as a word explanation about the syntax, right?

Reply:
@DavidAnson what do you think of these changes? Is there anything that concerns you that we can discuss?

Reply:
As I wrote above, I think "*/**" is wrong and I think the fact this example is shown on the next two lines is probably already good. I think the phrase "lint ... recursively" may be confusing as there is no recursion from the user's perspective.

Reply:
@DavidAnson okay, thank you for your sharing!
Here are my thoughts, that come from the issue https://github.com/igorshubovych/markdownlint-cli/issues/292
When I tried to search for this logic in the readme I used the `cmd+f` search on the page with the following `wildcard` and `recursively`. 

I don't quite understand why `I think "*/**" is wrong` because here https://github.com/igorshubovych/markdownlint-cli/issues/292#issuecomment-1146696384 this is precisely what was mentioned

Reply:
@DavidAnson If you still think this PR has no sense, and it's not discussable please let me know - I will close it :)

Reply:
Please read the comment you link to again, it does not say what you think it says.

Reply:
@DavidAnson `You're missing the first part of the folder wildcard if you want it have a recursive glob "**/*.md"` ? taken from https://github.com/igorshubovych/markdownlint-cli/issues/292#issuecomment-1146696384

Reply:
The existing examples and Nick's comment use the syntax `**/*.md` which is the conventional way to match all files with a particular extension. Your proposed change uses the syntax `*/**.md` which means something different.

Reply:
@DavidAnson ah, sorry, it was a typo. I didn't notice it. I thought to add `*/**.md`, but mistakenly I added `**/*.md`. Thank you for your look, I updated the PR accordingly

Please, let me know if it works for you :)